ironfish/src/network/peerNetwork.ts

```diff
@@ -269,6 +269,24 @@ export class PeerNetwork {
     this.started = true
 
     // Start the WebSocket server if possible
+    this.startWebSocketServer()
+
+    // Start up the PeerManager
+    this.peerManager.start()
+
+    // Start up the PeerConnectionManager
+    this.peerConnectionManager.start()
+
+    this.updateIsReady()
+
+    // Connect to bootstrap nodes
+    this.connectToBootstrapNodes()
+
+    // Connect to prior websocket outbound connections that were saved to disk
+    this.connectToPriorWebsocketConnections()
```

Comment:
Minor nit: these comments aren't adding anything since they're describing the function names. They made some sense when the code was one big jumble so it made it easier to see where the different logical chunks were, but now they're unnecessary

```suggestion
    this.startWebSocketServer()
    this.peerManager.start()
    this.peerConnectionManager.start()

    this.updateIsReady()

    this.connectToBootstrapNodes()
    this.connectToPriorWebsocketConnections()
```